var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -45,15 +46,16 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     )
 
     variant("opencl", default=False, description="Enable OpenCL backend")
+    variant("mpi_f08", default=False, description="Use mpi F08 module")
```

Comment:
This looks unused. What is it for?

Reply:
I forgot the cmake flag. Let me correct it.

Reply:
fixed